app/javascript/src/Metrics/components/ProductIndexPage.jsx

```diff
@@ -0,0 +1,67 @@
+// @flow
+
+import * as React from 'react'
+
+import { IndexPage } from 'Metrics'
+import { createReactWrapper } from 'utilities/createReactWrapper'
+import {
+  List,
+  ListItem
+} from '@patternfly/react-core'
+
+import type { Metric } from 'Types'
+
+type Props = {
+  applicationPlansPath: string,
+  createMetricPath: string,
+  mappingRulesPath: string,
+  metrics: Array<Metric>,
+  metricsCount: number
+}
+
+const ProductIndexPage = ({
+  applicationPlansPath,
+  createMetricPath,
+  mappingRulesPath,
+  metrics,
+  metricsCount
+}: Props): React.Node => {
+  const currentTabKey = (new URL(window.location.href)).searchParams.get('tab')
+  return (
+    <IndexPage
+      metrics={metrics}
+      metricsCount={metricsCount}
+      infoCard={currentTabKey === 'metrics' ? (
+        <>
+        Metrics track API usage. Metrics have these features:
+        <List>
+          <ListItem><i><b>Hits</b></i> is the 3scale native metric to which all methods report. This metric tracks the number of calls made to your API.</ListItem>
```

Comment:
```suggestion
          <ListItem><b>Hits</b> is the 3scale native metric to which all methods report. This metric tracks the number of calls made to your API.</ListItem>
```

Let's just stick to one font type, in this case bold, to indicate that Hits is a proper name in the context of 3scale.These font variations can make text more difficult to read—and each new variation requires some orientation by the user.